src/client/components/login-form.js

```diff
@@ -0,0 +1,78 @@
+import React, { Component } from "react";
+class LoginForm extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      email: "",
+      password: "",
+      result: ""
+    };
+  }
+  clickHandlerCancel = event => {
+    this.setState({
+      email: "",
+      password: ""
+    });
+  };
+  handleChange = event => {
+    this.setState({
+      [event.target.name]: event.target.value
+    });
+  };
+  handleSubmit = event => {
+    event.preventDefault();
+    try {
+      fetch("http://localhost:3000/api/auth/login/ ", {
```

Comment:
We already have that env variable!

Just use:
```
fetch(`${process.env.API_PATH}/auth/login/`, {
```

Reply:
I am getting Undefined when I am using this path. Can you pls check once

Reply:
Fixed it.